LEAF_Request_Portal/sources/Form.php

```diff
@@ -2842,36 +2842,433 @@ public function changeFormType($recordID, $categories)
         return 1;
     }
 
+ /**
+     * @param array $query - this is the decoded string used for query
+     * @return bool
+     */
+    private function isProcessQuery(array $query): bool
```

Comment:
```suggestion
 /**
     * isLargeQuery determines whether a query could potentially return a large result
     * @param array $query - this is the decoded string used for query
     * @return bool
     */
    private function isLargeQuery(array $query): bool
```

Rename to align with intended behavior.